### ISSUE TITLE
Trouble installing PicoBoot (DOL-101 USA)

### ISSUE THREAD

**Issue by MaltMiltank** [webhdx/PicoBoot#87]
| Answer some general questions first                             	  |  	     |
|:-------------------------------------------------------------------	|:------:|
| Which firmware version are you using?                             	| `v0.2` |
| Are you using the official Raspberry Pi Pico board?                 | yes |

I'm trying to install PicoBoot on a 101 I have. Seemingly no matter what I do, I get no output with the Pico connected, though the light on the GC and Pico turn on. I can successfully disconnect everything and have the GC boot like normal, but with the Pico connected, I get nothing. Triple-checked that my connections are correct. I got ahold of a 001 from my sibling and installed it perfectly fine on my first try using the same exact Pico and the same exact firmware, so I'm wondering if I'm missing a step for 101 installation? Pictures below:
![image](https://user-images.githubusercontent.com/61948359/234698150-c01b1279-90b0-4d39-8027-956f4dd45ce5.png)
![image](https://user-images.githubusercontent.com/61948359/234698223-50c80339-a3bf-4660-8ede-b4f95736e721.png)
![image](https://user-images.githubusercontent.com/61948359/234698269-aca9508f-853e-4ed4-9bba-64641a168fdb.png)

**Comment by MaltMiltank** [comment]
Update, I got it to work! Turned out I burned the PCB pretty bad under the blue wire connected to the chip. No idea how it booted fine after the first time I de-soldered everything, but after that it appeared the GC was toast. Managed to fix it in the most nasty-looking way possible, (scratching off the solder mask and extending the leg with solder) but it does work now!
![image](https://user-images.githubusercontent.com/61948359/234740923-5747923a-f66a-42ad-8675-fd6ca19fe616.png)
